### PR TITLE
Development

### PR DESCRIPTION
**Pull request by akshay-ap**

### CODE REVIEW

File: src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/AbstractAdapter.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *******************************************************************************/
+package blockchains.iaas.uni.stuttgart.de.plugin.bitcoin;
+
+import blockchains.iaas.uni.stuttgart.de.api.interfaces.BlockchainAdapter;
+import blockchains.iaas.uni.stuttgart.de.api.interfaces.FinalityConfidenceCalculator;
+
+public abstract class AbstractAdapter implements BlockchainAdapter {
```
Comment (ghareeb-falazi) [inline src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/AbstractAdapter.java]: I think it doesn't sense to have this class when we have only one adapter per plugin. Therefore, please remove it and move its contents to the `BitcoinAdapter` class

File: src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/BitcoinConnectionProfile.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
+package blockchains.iaas.uni.stuttgart.de.plugin.bitcoin; /*******************************************************************************
```
Comment (ghareeb-falazi) [inline src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/BitcoinConnectionProfile.java]: The package declaration should be placed after the copyright/license comment

File: src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/BitcoinConnectionProfile.java [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+ * Copyright (c) 2019 Institute for the Architecture of Application System - University of Stuttgart
```
Comment (ghareeb-falazi) [inline src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/BitcoinConnectionProfile.java:2]: If you introduced any sort of changes to this file, please updated the year and add yourself as a co-author

File: src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/BitcoinPlugin.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
+package blockchains.iaas.uni.stuttgart.de.plugin.bitcoin;
```
Comment (ghareeb-falazi) [inline src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/BitcoinPlugin.java:12]: Please add a copyright/license comment before the package

File: src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/BitcoinUtils.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package blockchains.iaas.uni.stuttgart.de.plugin.bitcoin; /*******************************************************************************
```
Comment (ghareeb-falazi) [inline src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/BitcoinUtils.java]: same as before: package after comment

**Review by ghareeb-falazi — CHANGES_REQUESTED** [review]

File: src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/BitcoinConnectionProfile.java (hunk 3 above)
Comment (akshay-ap) [reply src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/BitcoinConnectionProfile.java:2]: I did not change this class. But, I will add myself as co-author in other classes.

**Review by ghareeb-falazi — APPROVED** [review]
lgtm

**Pull request merged**